back/app/models/concerns/post.rb

```diff
@@ -7,7 +7,7 @@ module Post
   include GeoJsonHelpers
   extend ActiveSupport::Concern
 
-  PUBLICATION_STATUSES = %w[draft published].freeze
+  PUBLICATION_STATUSES = %w[draft submitted published].freeze
```

Comment:
I'm sorry if I misunderstood, but from our discussions I concluded that we would keep the published status, which was also written down in the ticket: https://www.notion.so/govocal/BE-Review-publication-statuses-ceb93867b7d84cd39af73a19ed9caf60?pvs=4#6c106cc83fca4411a6a323f90f66a649

There is indeed some duplication of information, but I thought we accepted this because we would like to ideally reimplement the reviewing as publication statuses and get rid of the prescreening status at some point in the future. I'm not sure if there could also be an impact on performance: it's probably easier and potentially faster to look at the publications status when filtering query results than to join the idea status (which might not exist for some inputs -- the survey results) and filter on the status code. The logic also felt better encapsulated that way, as the code does not need to make assumptions about which idea statuses are publicly visible.

Reply:
Alright, thanks for explaining